src/main/kotlin/org/stellar/walletsdk/util/Operation.kt

```diff
@@ -0,0 +1,29 @@
+package org.stellar.walletsdk.util
+
+import org.stellar.sdk.KeyPair
+import org.stellar.sdk.SetOptionsOperation
+import org.stellar.sdk.Signer
+import org.stellar.walletsdk.AccountSigner
+
+fun addSignerOperation(signer: AccountSigner): SetOptionsOperation {
+  val signerKeypair = KeyPair.fromAccountId(signer.address)
+  val signerKey = Signer.ed25519PublicKey(signerKeypair)
+
+  return SetOptionsOperation.Builder().setSigner(signerKey, signer.weight).build()
+}
+
+fun setThresholdsOperation(low: Int, medium: Int, high: Int): SetOptionsOperation {
+  return SetOptionsOperation.Builder()
+    .setLowThreshold(low)
+    .setMediumThreshold(medium)
+    .setHighThreshold(high)
+    .build()
+}
+
+fun setMasterKeyWeightOperation(weight: Int): SetOptionsOperation {
+  return SetOptionsOperation.Builder().setMasterKeyWeight(weight).build()
+}
+
+fun lockMasterKeyOperation(): SetOptionsOperation {
+  return setMasterKeyWeightOperation(0)
+}
```

Comment:
I'm not convinced this is useful either, its just wrapping existing object from the Java SDK.

Reply:
These are helper functions to make the maintainer developer experience better. That's why they are in the `util` module and not part of the "main" module. I'm used to splitting everything into smaller chunks; should we not do that in the Wallet SDK? How should we handle helper functions that are not part of the API?

Reply:
I think in concept these are useful in that the API is simpler than Java SDK's.

For ex: Calling `lockMasterKeyOperation` is more intuitive than calling `setMasterKeyWeightOperation(0)`.

But there is definitely a fine line between when it's useful and when it's just wrapping. For ex, maybe `setThresholdsOperation` isn't as useful because you're not really obfuscating anything "tricky" here